defaults/main.yml

```diff
@@ -16,6 +16,8 @@ he_local_vm_dir_prefix: localvm
 he_appliance_ova: ''
 he_root_ssh_pubkey: ''
 he_root_ssh_access: 'yes'
+he_openscap_profile: ''
+he_openscap_datastream: ''
```

Comment:
please add to these 2 variables to README.md

Reply:
Fixed - added a single variable he_apply_openscap_profile